asterisk_click2dial/asterisk_click2dial.py

```diff
@@ -201,41 +370,75 @@ def _connect_to_asterisk(self, cr, uid, context=None):
         # Note : if I write 'Error' without ' :', it won't get translated...
         # I don't understand why !
 
-        ast_server = self._get_asterisk_server_from_user(cr, uid, user, context=context)
+        ast_server = self._get_asterisk_server_from_user(
+            cr, uid, user, context=context
+        )
         # We check if the current user has a chan type
         if not user.asterisk_chan_type:
-            raise osv.except_osv(_('Error :'), _('No channel type configured for the current user.'))
+            raise osv.except_osv(
+                _('Error :'),
+                _('No channel type configured for the current user.')
+            )
 
         # We check if the current user has an internal number
         if not user.resource:
-            raise osv.except_osv(_('Error :'), _('No resource name configured for the current user'))
-
-
-        _logger.debug("User's phone : %s/%s" % (user.asterisk_chan_type, user.resource))
-        _logger.debug("Asterisk server = %s:%d" % (ast_server.ip_address, ast_server.port))
+            raise osv.except_osv(
+                _('Error :'),
+                _('No resource name configured for the current user')
+            )
+
+        _logger.debug(
+            "User's phone : %s/%s" % (user.asterisk_chan_type, user.resource)
+        )
+        _logger.debug(
+            "Asterisk server = %s:%d"
+            % (ast_server.ip_address, ast_server.port)
+        )
 
         # Connect to the Asterisk Manager Interface
         try:
-            ast_manager = Manager.Manager((ast_server.ip_address, ast_server.port), ast_server.login, ast_server.password)
+            ast_manager = Manager.Manager(
+                (ast_server.ip_address, ast_server.port),
+                ast_server.login,
+                ast_server.password
+            )
         except Exception, e:
-            _logger.error("Error in the Originate request to Asterisk server %s" % ast_server.ip_address)
-            _logger.error("Here is the detail of the error : '%s'" % unicode(e))
-            raise osv.except_osv(_('Error :'), _("Problem in the request from OpenERP to Asterisk. Here is the detail of the error: '%s'" % unicode(e)))
-            return False
+            _logger.error(
+                "Error in the Originate request to Asterisk server %s"
+                % ast_server.ip_address
+            )
+            _logger.error(
+                "Here is the detail of the error : '%s'" % unicode(e)
```

Comment:
don't use '%' in calls to logger methods

and this will fail horribly if there is a non ascii char in unicode(e)